packages/node-integration-tests/utils/index.ts

```diff
@@ -291,12 +304,14 @@ export class TestEnv {
 
         nock.cleanAll();
 
-        this.server.close(() => {
+        void this._closeServer().then(() => {
           resolve(reqCount);
         });
-
-        resolve(reqCount);
       }, options.timeout || 1000);
     });
   }
+
+  private _closeServer(): Promise<void> {
+    return this._terminator.terminate();
```

Comment:
just to be clear, do we not need to call `this.server.close()` anymore, is that handled by the terminator?

Reply:
Yes, it's handled by it -> https://github.com/gajus/http-terminator/blob/aabca4751552e983f8a59ba896b7fb58ce3b4087/src/factories/createInternalHttpTerminator.ts#L156

Reply:
👍